libraries/AP_Scripting/tests/serial_loopback.lua

```diff
@@ -0,0 +1,36 @@
+local ser_driver = serial:find_serial(0)
+local ser_device = serial:find_simulated_device(28, 0)
```

Comment:
it would be nice to not tie this method to just the SCR_SER devices. I've wanted for a while to be able to attach to serial devices that don't have their protocol set to scripting. For example, when writing a rangefinder driver in lua it would be nice for the user to be able to set the protocol to rangefinder instead of scripting.

Reply:
I don’t understand how that relates. That could be a future improvement, but we would have to figure out how to disconnect the existing driver I think?